src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+const total = document.querySelector('.total-population');
+const aver = document.querySelector('.average-population');
```

Comment:
```suggestion
const averageElement = document.querySelector('.average-population');
```
Не сокращай названия